src/coreclr/vm/ilstubresolver.cpp

```diff
@@ -344,7 +344,7 @@ ILStubResolver::AllocGeneratedIL(
     if (!UseLoaderHeap())
     {
         NewArrayHolder<BYTE>             pNewILCodeBuffer = new BYTE[cbCode];
-        NewArrayHolder<CompileTimeState> pNewCompileTimeState = (CompileTimeState*)new BYTE[sizeof(CompileTimeState)];
+        NewHolder<CompileTimeState>      pNewCompileTimeState = new CompileTimeState{};
```

Comment:
For my education - does `new CompileTimeState{}` allocate the space without calling any constructor?

Reply:
Good catch. It actually zero-inits, so I can remove the `memcpy` on the next line.